includes/admin/payments/class-order-items-table.php

```diff
@@ -249,13 +254,11 @@ public function column_name( $order_item ) {
 		$order_item_title = '<strong><a class="row-title" href="' . add_query_arg( array(
 				'action' => 'edit',
 				'post'  => $order_item->product_id,
-			), admin_url( 'post.php' )  ) . '">' . $order_item->get_order_item_name() . '</a>' . $state . '</strong>';
-
-		// See what the current name of the product is:
-		$current_product_name = edd_get_download_name( $order_item->product_id, $order_item->price_id );
-		if ( strtolower( htmlentities( $current_product_name ) ) !== strtolower( htmlentities( $order_item->get_order_item_name() ) ) ) {
-			$status_help = '<p>' . sprintf( __( 'This product has been renamed since this purchase. It is now %s.', 'easy-digital-downloads' ), $current_product_name );
+		), admin_url( 'post.php' )  ) . '" aria-label="' . $order_item->get_order_item_name() . esc_attr( true === $changed_name ? ( ': ' . $status_help ) : '' ) . '">';
```

Comment:
```suggestion
		), admin_url( 'post.php' )  ) . '" aria-label="' . esc_attr( $order_item->get_order_item_name() . ( true === $changed_name ? ( ': ' . $status_help ) : '' ) ) . '">';
```

Also, while you're here, can you `esc_url()` on the `href` value?